etc/meteo-qt.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for meteo-qt
+# Description: System tray application for weather status information
+# This file is overwritten after every install/update
+# Persistent local customizations
+include meteo-qt.local
+# Persistent global definitions
+include globals.local
+
+# Allow python (blacklisted by disable-interpreters.inc)
+noblacklist ${PATH}/python3*
+noblacklist /usr/lib/python3*
+
+noblacklist ${HOME}/.config/autostart
+noblacklist /usr/share/applications/meteo-qt.desktop
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/meteo-qt
+whitelist ${HOME}/.config/meteo-qt
+whitelist ${HOME}/.config/autostart
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+nodbus
```

Comment:
Doesn't break tray-icon?

Reply:
tray icon is working fine (for me at least)

Reply:
Confirming tray icon works fine with `nodbus`.